Network/Mail/SMTP.hs

```diff
@@ -129,18 +132,19 @@ connectStream getMailHostName st = do
         msg <- tryCommand 3 (SMTPC st []) (HELO $ B8.pack senderHost) 250
         return (SMTPC st (tail $ B8.lines msg))
 
-parseResponse :: Handle -> IO (ReplyCode, ByteString)
-parseResponse st = do
-    (code, bdy) <- readLines
-    return (read $ B8.unpack code, B8.unlines bdy)
+parseResponse :: Conn.Connection -> IO (ReplyCode, ByteString)
+parseResponse conn = do
+  (code, bdy) <- readLines
+  return (read $ B8.unpack code, B8.unlines bdy)
```

Comment:
I'd prefer if indentation-only changes weren't committed to preserve the git history.